integration-test-suite/cds-toolkit-test-framework/src/main/groovy/org/wso2/cds/test/framework/constant/AUConstants.groovy

```diff
@@ -590,7 +590,7 @@ class AUConstants extends OBConstants {
     public static final String CALLBACK_MISMATCH = "Callback url mismatch"
     public static final String LOCALHOST = "localhost"
 
-    public static String REPORTING_DBNAME = "openbank_ob_reporting_statsdb"
+    public static String REPORTING_DBNAME = "mig_openbank_ob_reporting_statsdb"
```

Comment:
Why is this changed? Shall we move this to the config xml since it can change?

Reply:
Fixed. This was changed due to internal testing purpose. Revert the change.